bugzoo/mgr/container.py

```diff
@@ -576,7 +576,7 @@ def command(self,
 
         output = []
         for line in out:
-            line = line.decode('utf-8').rstrip('\n')
+            line = line.decode('utf-8',"backslashreplace").rstrip('\n')
```

Comment:
Minor nitpick: can you add a space between the comma and the new argument? Also, could you use `'` rather than `"`?

Reply:
Done!